haskell-src/Concordium/Types/Transactions.hs

```diff
@@ -167,7 +168,7 @@ makeAccountTransaction atrSignature atrHeader atrPayload = AccountTransaction{..
   where
     atrSignHash = transactionSignHashFromHeaderPayload atrHeader atrPayload
 
--- | @SPEC: <$DOCS/Transactions#serialization-format-transactions>
+-- | @SPEC: <$DOCS/Transactions#serialization-format-v1-transactions>
```

Comment:
Why this change? Do we still actually have the document that this is a reference to?

Reply:
I thought this was a way of referencing a section in the bluepaper for internal purposes or something. If this is not the case I'll remove them. 😅